### DISCUSSION TITLE
How do I call a q-dialog inside a q-select?

### DISCUSSION THREAD

**Discussion by hzlgrce** [quasarframework/quasar#10982] [General - Components / Directives / etc] [answered]
i'm not sure if im doing this the right way, but i have a custom dialog which i want to call when i press the q-select. i have tried "@ click= openDialog()" but its not showing up. is this even possible?

**Comment by MilosPaunovic** [comment, marked as answer]
You can check `Events` tab of the [QSelectAPI](https://quasar.dev/vue-components/select#qselect-api) and attach a method for dialog opening to one of 'em.

**Reply by hzlgrce** [reply]
thank you! i just used @popup-show an it worked!
